opae.spec

```diff
@@ -9,7 +9,7 @@ Group:          Development/Libraries
 Vendor:         Intel Corporation
 Requires:       uuid, json-c, python3
 URL:            https://github.com/OPAE/%{name}-sdk
-Source0:        https://github.com/OPAE/opae-sdk/releases/download/%{version}-%{Release}/%{name}-%{version}.tar.gz
```

Comment:
Why can't we use %{Release}? It's set to 2 (or dist) in line 4.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1806017 

https://github.com/trixirt/opae-rpm/blob/opae-1.4.0-5/opae.spec

https://fedoraproject.org/wiki/Packaging:DistTag


Tom was updated spec file like below,  followed his  pattern

Version:        1.4.0
Release:        5%{?dist}
Source0:        https://github.com/OPAE/opae-sdk/releases/download/%{version}-1/%{name}-%{version}-1.tar.gz

Reply:
This seems like a step backward in terms of maintainability.  I don't see why we would need to hard-code things and require that so many changes be made each time.

Reply:
Yes, hardcoding for a release would work to but this doesn't mean we can't use a variable.
We are already setting it in line 4. Using the variable should work just as well as using the literal value except that now we only have to change it in once place if/when we have to change it.

Reply:
set %{Release}? to 2 in spec file